packages/renderer/src/lib/image/BuildImageFromContainerfile.svelte

```diff
@@ -323,13 +323,20 @@ $effect(() => {
   }
   buildImageInfo.selectedProvider = selectedProvider;
 });
+
+let isImageNameLowercase = $derived(
+  buildImageInfo.containerImageName === buildImageInfo.containerImageName.toLowerCase(),
+);
 let hasInvalidFields = $derived(
   !buildImageInfo.containerFilePath ||
     !buildImageInfo.containerBuildContextDirectory ||
     (platforms.length > 1 && !buildImageInfo.containerImageName) ||
     platforms.length === 0 ||
+    !isImageNameLowercase ||
```

Comment:
suggestion: we can probably remove `isImageNameLowercase` and have only `errorContainerImageName` as error mean invalid


```suggestion
    !errorContainerImageName ||
```